Tuning/parallel_optimization.py

```diff
@@ -0,0 +1,62 @@
+import sys
+
+sys.path.append("../")
+
+import optuna  # noqa: E402
+from pathos.multiprocessing import ProcessingPool  # noqa: E402
+from stable_baselines3 import PPO  # noqa: E402
+from stable_baselines3.ppo.policies import MlpPolicy  # noqa: E402
+
+from Hack import load, rl  # noqa: E402
+
+
+def create_train_model(price_array, learning_rate):
+    """
+    Creates and trains the deep learning model
+    """
+    start_idx = 0
+    end_idx = 4 * 2 * 24 * 7  # start_of_2020 # 2019->2020 # 2*24*7
+    obs_price_array = price_array[start_idx:end_idx]
+    power = 0.5
+    env = rl.energy_price_env(obs_price_array, window_size=24 * 2, power=power)
+    model = PPO(MlpPolicy, env, verbose=0, learning_rate=learning_rate)
+    model.learn(100)
+    return model
+
+
+def evaluate_model(iter_obj):
+    val_list = []
```

Comment:
```suggestion
    total_val = 0
```

Making this quicker too, take out the lists which are slow